README.md

```diff
@@ -83,14 +89,17 @@ sudo apt -y install libsgx-enclave-common libsgx-dcap-ql libsgx-dcap-ql-dev libs
 curl -sSL https://get.docker.com/ | sh
 ```
 
-## Download Mystikos
+## 2. Download and Install Mystikos
+
+### Download and Install from GitHub
 
 You can [download the latest build here](https://github.com/deislabs/mystikos/releases)
 then simply decompress it, add it to your path, and run it.
 
+**Tarball installation**
 ```
 # change this to match the latest version
-LATEST='0.5.0'
+LATEST='0.7.0'
```

Comment:
You have renamed the packages to "Ubuntu20 etc", so you have to update these docs.
The best thing might be to split into two pages and add links to each.